docs/src/content/docs/reference/configuration.mdx

```diff
@@ -183,6 +183,7 @@ sidebar: [
 type SidebarItem = {
 	label: string;
 	translations?: Record<string, string>;
+	badge?: string | BadgeConfig;
```

Comment:
I cannot suggest a change for this but I think the `badge?: string | BadgeConfig;` right below this one can be removed now.

Reply:
Completely missed that one. Great catch!